nimbus/helm/templates/nimbus-deployment.yaml

```diff
@@ -0,0 +1,95 @@
+{{- range $key, $nimbusClient := .Values.nimbus.clients }}
```

Comment:
If the array is named nimbus.clients, the variable can be $client with the nimbus to make it consistent.

From our previous discussion, it's also fine to keep it as it is if you think nimbusClient makes more sense semantically.

Reply:
changed it to `$client`